README.md

```diff
@@ -190,87 +189,49 @@ Files for the local database e2e-tests are located in the `./end-to-end-test/loc
 
 ### Running `localdb` e2e-tests for development
 
-1. Start cBioPortal (including session service) using the [cBioPortal docker compose solution](https://docs.cbioportal.org/2.1.1-deploy-with-docker-recommended/docker#quick-start).
-cBioPortal must be accessible on _http://localhost:8080_.
-
-```
-cd
-git clone https://github.com/cBioPortal/cbioportal-docker-compose.git
-cd cbioportal-docker-compose
-./init.sh
-docker-compose up -d
-```
+1. You need to have Docker installed and running.
 
-:warning: If the frontend requires a specific backend version, make sure to deploy this instead of the default version supported by the docker compose solution.
+2. You need to have the [jq](https://stedolan.github.io/jq/) package installed on your system. E.g. using brew:
+   ```brew install jq```
 
-2. Install gene panels and gene sets for study_es_0:
+In a terminal, start the frontend dev server
 
 ```
-cd ~/cbioportal-docker-compose
-docker-compose run --rm cbioportal sh -c '
-    cd /cbioportal/core/src/main/scripts/ \
-    && ./importGenePanel.pl --data /cbioportal/core/src/test/scripts/test_data/study_es_0/data_gene_panel_testpanel1.txt \
-    && ./importGenePanel.pl --data /cbioportal/core/src/test/scripts/test_data/study_es_0/data_gene_panel_testpanel2.txt \
-    && ./importGenesetData.pl --data /cbioportal/core/src/test/resources/genesets/study_es_0_genesets.gmt --new-version msigdb_6.1 \
-    && ./importGenesetHierarchy.pl --data /cbioportal/core/src/test/resources/genesets/study_es_0_tree.yaml'
+export BRANCH_ENV=custom
+yarn install --frozen-lockfile // only necessary first time
+yarn buildDLL:dev // only necessary first tiem
```

Comment:
prolly good to use `#` here for comment, cause then you can copy+paste them (`//` is not a comment in bash)